src/renderers/shaders/ShaderChunk/global_fragment.glsl

```diff
@@ -0,0 +1 @@
+// template for injecting varying, uniform and functions above main below other includes
```

Comment:
@mrdoob i guess with `GLOBAL_VERTEX` and `GLOBAL_FRAGMENT` this now does the exact same thing as https://github.com/mrdoob/three.js/issues/11475

so ill close it